client/app/stories/Styleguide.stories.jsx

```diff
@@ -0,0 +1,358 @@
+  .add('Error', () => (
+    <div className={`${css.errorField} error`}>
+      <h2 className={`${css.errorText} error`}>errors</h2>
+      <ul className={`${css.errorText} error`}>
+        <li>this</li>
+        <li>is</li>
+        <li>an</li>
+        <li>example</li>
+      </ul>
+    </div>
+  ))
+  .add('Forms', () => (
+    <form>
+      <label htmlFor="label" className={`${css.label} label`}>
+        label
+        <br />
+        <input
+          style={{ marginBottom: '10px' }}
+          type="text"
+          className={`${css.inputStyle} inputStyle`}
+          placeholder="inputStyle"
+        />
+        <br />
+        <textarea />
+      </label>
+    </form>
+  ))
+  .add('Grids', () => (
+    <Fragment>
+      <div
+        className={`${css.gridTwo} gridTwo`}
+      >
+        <div
+          className={`${css.gridTwoItemBoxDark} gridTwoItemBoxDark`}
+          style={{ border: '2px solid' }}
+        >
+              gridTwoItem
+        </div>
+        <div
+          className={`${css.gridTwoItemBoxDark} gridTwoItemBoxDark`}
+          style={{ border: '2px solid' }}
+        >
+            gridTwoItem
+        </div>
+      </div>
+      <div className={`${css.gridThree} gridThree`}>
+        <div
+          className={`${css.gridThreeItemBoxGhost}  gridThreeItemBoxGhost`}
+          style={{ border: '2px solid' }}
+        >
+            gridThreeItem
+        </div>
+        <div
+          className={`${css.gridThreeItemBoxGhost}  gridThreeItemBoxGhost`}
+          style={{ border: '2px solid' }}
+        >
+            gridThreeItem
+        </div>
+        <div
+          className={`${css.gridThreeItemBoxGhost}  gridThreeItemGhost`}
+          style={{ border: '2px solid' }}
+        >
+            gridThreeItem
+        </div>
+      </div>
+      <div
+        className={`${css.gridThree} gridThree`}
+        style={{ margin: '10px' }}
+      >
+        <div
+          className={`${css.gridThreeItemBoxLight}  gridThreeItemBoxLight`}
+          style={{ margin: '10px' }}
+        >
+          <h1 className={`${css.gridh1}  gridh1`}>gridItemBoxLight</h1>
+        </div>
+        <div
+          className={`${css.gridThreeItemBoxDark}  gridThreeItemBoxDark `}
+          style={{ margin: '10px' }}
+        >
+          <h1 className={`${css.gridh1}  gridh1`}>gridItemBoxDark</h1>
+        </div>
+        <div
+          className={`${css.gridThreeItemBoxGhost}  gridThreeItemBoxGhost`}
+          style={{ margin: '10px' }}
+        >
+          <h1 className={`${css.gridh1}  gridh1`}>gridItemBoxGhost</h1>
+        </div>
+      </div>
+    </Fragment>
+  ), { notes: { markdown: '# hi' } })
+  .add('Margins', () => (
+    <Fragment>
+      <div style={{
+        margin:'10px'
+        }}>
+        <div className={`${css.marginRight} marginRight`} style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}>
```

Comment:
We can store the following `style={{ display: 'inline-block', border: '1px solid', padding:'10px' }}` as:
```
const marginGroupStyle = {
  display: 'inline-block',
  border: '1px solid',
  padding:'10px',
};
```

```
<div className={`${css.marginRight} marginRight`} style={marginGroupStyle}>
```